iceoryx_posh/test/moduletests/test_gw_gateway_generic.cpp

```diff
@@ -146,12 +147,11 @@ TEST_F(GatewayGenericTest, HandlesMaxmimumChannelCapacity)
     // ===== Test
     for (auto i = 0U; i < iox::MAX_CHANNEL_NUMBER; i++)
     {
-        auto result =
-            sut->addChannel(iox::capro::ServiceDescription(
-                                iox::capro::IdString_t(iox::cxx::TruncateToCapacity, iox::cxx::convert::toString(i)),
-                                iox::capro::IdString_t(iox::cxx::TruncateToCapacity, iox::cxx::convert::toString(i)),
-                                iox::capro::IdString_t(iox::cxx::TruncateToCapacity, iox::cxx::convert::toString(i))),
-                            StubbedIceoryxTerminal::Options());
+        auto result = sut->addChannel(
+            iox::capro::ServiceDescription(iox::cxx::into<iox::capro::IdString_t>(iox::cxx::convert::toString(i)),
```

Comment:
Maybe it is too much for this PR. But how awesome would it be when you would add an `into` implementation for `uint64_t` then we could just write here:
```cxx
iox::capro::ServiceDescription(iox::cxx::into<iox::capro::IdString_t>(i),
```

and can get rid of `convert`.

Could you please add a todo for this.

Reply:
@elfenpiff I definitely agree, that would be brilliant! Let's work on this topic once we reach the refactoring of `convert`. I added a todo.